### PR TITLE
Small change to ad placeholder tag design

### PR DESCRIPTION
**Pull request by zhouyx**
For #8261 
cc @spacedino @jasti

### CODE REVIEW

File: extensions/amp-ad/0.1/amp-ad.css [hunk 1]
```diff
@@ -51,7 +51,7 @@ amp-embed iframe {
   border-radius: 2px;
   color: #fff;
   font-size: 16px;
-  line-height: 16px;
+  line-height: 100%;
```
Comment (camelburrito) [inline extensions/amp-ad/0.1/amp-ad.css]: you should not need this

Comment (zhouyx) [inline extensions/amp-ad/0.1/amp-ad.css]: removed. so it is 100% by default?

File: extensions/amp-ad/0.1/amp-ad.css [hunk 2]
```diff
@@ -60,11 +59,11 @@ amp-embed iframe {
 .i-amphtml-ad-default-holder[experiment2]:after {
   font-size: 10px;
   line-height: 10px;
```
Comment (camelburrito) [inline extensions/amp-ad/0.1/amp-ad.css]: you can get rid of this too

File: extensions/amp-ad/0.1/amp-ad.css (hunk 1 above)
Comment (camelburrito) [inline extensions/amp-ad/0.1/amp-ad.css]: line-height is 1 by default (the unit is em) which means it is the same as the computed/set font-size of the element, 1em = 100% ,

**Pull request merged**